addons/common/XEH_postInit.sqf

```diff
@@ -86,6 +87,14 @@
     };
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(blockSpeaking), {
+    params ["_object", "_set"];
+    TRACE_2("blockSpeaking EH",_object,_set);
+    if (["acre_main"] call FUNC(isModLoaded)) then {
```

Comment:
I thought about adding the tfar setVariables here too instead of in `setVolume`, but I'm not familiar enough with it to know whether `tf_voiceVolume` is hearing or speaking volume

Reply:
I moved the code over so things are consistant between the 2 mods
I tested by watching `[player, true, "guy"] call TFAR_fnc_preparePositionCoordinates;` 
and verified that the output changes
ref https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/master/addons/core/functions/fnc_preparePositionCoordinates.sqf#L103
(although this version is different from the one on steam, but both seem to work the same)

Reply:
Nice, thanks!